chcemvediet/templates/main/homepage/didyouknow/main.html

```diff
@@ -8,6 +9,11 @@
  %
 {% endcomment %}
 
+{% block heading %}
```

Comment:
@martinmacko47 Nato aby som nadpis dal do pomocneho template som potreboval vytvorit novy block, pretoze template pozuiva `extend`.

Reply:
Lepsie by bolo to rozdelit na dva subory: V `main.html` bude nadpis a bude includovat subor `items.html`. A v `items.html` bude to co doteraz.

Do `snippets/carousel/main.html` by som `block heading` urcite nedaval. Kedze to ma byt reusable snippet. A pri inych pouzitiach nemusi heading davat vobec zmysel.

Reply:
takto to bude ovela lepsie, dakujem. :ok: